webpack/webpack.dev.config.js

```diff
@@ -102,6 +102,7 @@ module.exports = Merge.smart(commonConfig, {
       APPLE_APP_STORE_URL: 'https://www.apple.com/ios/app-store/',
       GOOGLE_PLAY_URL: 'https://play.google.com/store',
       ORDER_HISTORY_URL: 'localhost:1996/orders',
+      CYBERSOURCE_URL: 'https://localhost/cybersource',
```

Comment:
I'm guessing this means there's no way to test submission locally?

Reply:
The way to test the full payment flow locally is to point your local frontend app at stage.